nethserver-mail2.spec

```diff
@@ -28,6 +28,7 @@ Requires: nethserver-dnsmasq, nethserver-unbound
 Requires: rspamd
 Requires: redis
 Requires: zstd
+Requires: mod_authnz_pam
```

Comment:
Wasn't it already included by httpd-admin?

Reply:
I'd leave this requires also here since it doesn't require httpd-admin.

Reply:
Well, it actually does it, as it extends its configuration... 

however yes, it's required because httpd-admin implements PAM authentication with a wrapper script.

Reply:
ok